rollout/trafficrouting/alb/alb.go

```diff
@@ -129,9 +132,21 @@ func (r *Reconciler) VerifyWeight(desiredWeight int32, additionalDestinations ..
 		r.cfg.Status.ALB = nil
 		return nil, nil
 	}
+
+	if !rolloututil.ShouldVerifyWeight(r.cfg.Rollout) {
```

Comment:
I see we have two functions with the same name: shouldVerifyWeight it's confusing.
`rolloututil.ShouldVerifyWeight` is based on a step, rs etc.
`shouldVerifyWeight` is a configuration value. 

I think we have to rename one of that